extensions/wikia/AdEngine/js/lookup/rubicon/rubiconVulcan.js

```diff
@@ -10,19 +10,21 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconVulcan', [
 	'use strict';
 
 	var accountId = 7450,
+		incontentAdSettings = {
+			siteId: 55412,
+			size: [640, 480],
```

Comment:
Please confirm that we should use 640x480 instead of 320x480 for MOBILE_IN_CONTENT